include/dev/RedundantADC.hpp

```diff
@@ -0,0 +1,60 @@
+#pragma once
+
+#include <EVT/io/ADC.hpp>
+
+namespace IO = EVT::core::IO;
+
+namespace HIB::DEV {
+
+/**
+ * This class allows processing readings from redundant ADCs and checking for errors.
```

Comment:
This is a nitpick, but don't end any of your lines with periods
```suggestion
 * This class allows processing readings from redundant ADCs and checking for errors
```